.github/workflows/c.yaml

```diff
@@ -143,7 +137,21 @@ jobs:
                     xdelta3 libjpeg-turbo-progs
                 ;;
             esac
+            $sudo python${pyver} -m pip install --upgrade pip
+            $sudo python${pyver} -m pip install meson 
         esac
+        libdicom_url=https://github.com/jcupitt/libdicom.git
+        libdicom_branch=cleanup
+        if ! git clone -b $libdicom_branch $libdicom_url; then
+          exit 1
+        fi
+        cd libdicom
+        if ! meson setup builddir; then
+          cat builddir/meson-logs/meson-log.txt
+          exit 1
+        fi
+        ninja -C builddir
+        $sudo ninja -C builddir install
```

Comment:
Aside from this block, I think all of the changes in this file are superfluous?  We'll need macOS CI, and we need to support the Meson version that ships natively with each distro.  If there are changes in this file that you feel are necessary, let's discuss.

Nit: indentation should be four spaces.

Reply:
Yes, it's still a draft while I learn the meson wrap system. I'll clean it up and squash it before making it final.

Reply:
Is the plan to ship a libdicom wrap in OpenSlide, or to have CI explicitly build libdicom?

Also, it looks like you're continuing to work on having CI upgrade Meson in the various distros.  CI should test what we actually need to support, which is the Meson version shipped by the distros.  What problem are you trying to solve there?

Reply:
libdicom needs a more recent meson than the version shipped in at least centos8 and ubuntu 22.04.

Reply:
... I was thinking of including a libdicom wrap, yes.

Reply:
https://github.com/ImagingDataCommons/libdicom/pull/47 reduces libdicom's minimum Meson version to older than OpenSlide's minimum.

Reply:
And https://github.com/ImagingDataCommons/libdicom/pull/49 is needed to find uthash correctly on Ubuntu 20.04.